docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
 
-## Transactions 
+Validator Operators are free to set their own **commission rate** to charge for delegating. The **commission rate** can change at most once daily. However, a validator's **commission rate** CANNOT exceed the **maximum commission rate** set when the validator node was first initiated. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
 
-How to view the mempool? Or pending transactions 
-If you're a validator you can keep your mempool to yourself when it is your turn to propose a block then you can submit your own transactions. 
+![validator-details](@site/static/img/validator-details.png)
 
-If you sync your own node with other nodes then you can view pending transactions.
+### Redelegation
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+1. A **Delegator** can **redelegate** tokens from **Validator A** to **Validator B** without incurring any penalties or extra waiting times. A redelegation maturation period now follows for 21 days. 
 
-* We have a min-gas-price parameter set at 0.000005 CUDOS below which txs won’t be accepted into mempools
+2. Following on from the above example, if a **Delegator** wishes to **redelegate** their tokens from **Validator B** to a hypothetical **Validator C** they must wait *21 days*.
 
-    This parameter is local to each node on the network, can be changed off-chain, and cannot be queried on-chain.
+3. A **Validator Operator** can redelegate funds to an *existing* node in two ways:
 
-    Hence we must rely on the community for this implementation to work properly.
+- Connecting a wallet to the [**Cudos Dashboard**](https://dashboard.cudos.org/staking) and redelegating funds to a Validator node of their choice. 
 
-* Full-nodes that keep these mempools must have a mechanism to protect themselves from over-usage.
+- Using `cudos-noded` CLI to redelegate funds. 
 
-:::warning 
-Default `gas-wanted on the CLI is 200,000`. Hence the fee is calculated based on this. Users must be aware of this. Be careful to specify this as what you are prepared to pay. 
+:::caution
 
-Hence most transactions cost 1 CUDOS
+The delegated stake on a Validator node must be at least 2M CUDOS to continue running
 :::
 
-* Sometimes the CLI throws the “Error: RPC error -32603 - Internal error: timed out waiting for tx to be included in a block” error. In this case, the tx is most likely successful, and can be found in the Transactions tab on the explorer page.
-
-One workaround is to use the `--broadcast-mode` flag that returns only a tx hash once a tx is submitted.
+### Undelegation
 
-There was a ticket on making this the default behaviour, but looks like it has been discarded.
+A **Delegator** can choose to **undelegate** their tokens at any time. 
```

Comment:
Maybe we should add something like "In that case the funds will be released to the delegator 21 days later, following the date of the undelegation. During this time, the undelegated tokens are in the unbonding state, and consequently do not contrinute towards earning of rewards or voting power."

Reply:
done